bundleserver/src/main/java/net/discdd/server/BundleServerApplication.java

```diff
@@ -54,35 +54,41 @@ protected void refresh(ConfigurableApplicationContext applicationContext) {
                 super.refresh(applicationContext);
             }
         }
+
         var app = new MySpringApplication(BundleServerApplication.class);
-        if (args.length > 0) {
-            Resource resource = new FileSystemResource(args[0]);
-            if (resource.exists()) {
-                try {
-                    Properties properties = PropertiesLoaderUtils.loadProperties(resource);
-                    app.setDefaultProperties(properties);
-                    args = Arrays.copyOfRange(args, 1, args.length);
-                } catch (Exception e) {
-                    logger.log(SEVERE, "Please enter valid properties file path!");
-                    System.exit(1);
-                }
-            } else {
-                logger.log(SEVERE, "Entered properties file path does not exist!");
-                System.exit(1);
-            }
-        } else {
+
+        if (args.length == 0) {
             logger.log(SEVERE, "Please enter properties file path as argument!");
             System.exit(1);
         }
 
+        Resource resource = new FileSystemResource(args[0]);
+        if (!resource.exists()) {
+            logger.log(SEVERE, String.format("Entered properties file path %s does not exist!", args[0]));
+            System.exit(1);
+        }
+
+        try {
+            var properties = PropertiesLoaderUtils.loadProperties(resource);
+            app.setDefaultProperties(properties);
+            args = Arrays.copyOfRange(args, 1, args.length);
+        } catch (Exception e) {
+            logger.log(SEVERE, "Please enter valid properties file path!");
+            System.exit(1);
+        }
+
         app.setBanner((e, s, o) -> o.println(BananaUtils.bananansi("DDD Bundle Server", Font.ANSI_SHADOW, Ansi.GREEN)));
```

Comment:
Do we need the banner for CLI?

Reply:
that gets ignored because banner mode is set to off.